Makefile

```diff
@@ -144,6 +145,16 @@ distclean: clean
 test: $(BUILD) ../ErmrestDataUtils
 	node test/jasmine-runner.js
 
+# Rule to run the unit tests
+.PHONY: testsingle
+testsingle: $(BUILD) ../ErmrestDataUtils
+	node test/single-test-runner.js
+
+# Rule to run the unit tests
+.PHONY: testsingledebug
```

Comment:
Please remove this. node-debug package has been deprecated.

Reply:
If you want you can change it to `node --inspect-break` but I don't think this is useful and I don't see the point of having a debug make target.